files/messages/checkout.txt

```diff
@@ -1,13 +1,17 @@
 Hi everyone,
 
-Thanks again for taking part in instructor training this week. In order to wrap up, we would like you to do the steps outlined in http://swcarpentry.github.io/instructor-training/checkout/:
+Thanks again for taking part in instructor training this week. I really enjoyed spending time with you, and I look forward to your continued involvement with the Software and Data Carpentry Communities. 
+
+We'd be very grateful if you could take a couple of minutes to fill in this survey (https://www.surveymonkey.com/r/post-instructor-training) to provide annoymous feeback about the workshop.
+
+In order to wrap up, we would like you to do the steps outlined in http://swcarpentry.github.io/instructor-training/checkout/:
 
 1. Submit something for one of our lessons.
 
 2. Take part in an online discussion session.
 
 3. Teach a 5-minute demonstration lesson online.
 
-If you have any questions or suggestions about these steps, please contact your trainer or mail us at training-help@software-carpentry.org.  Please note that we would like everyone to complete all three steps within 90 days of the training course.  If you are not able to do this, please contact your trainer to discuss reasons and options.
+If you have any questions or suggestions about these steps, please contact me or all the trainers at [training-help@software-carpentry.org](mailto:training-help@software-carpentry.org?Subject=Instructor%20training%20checkout). Please note that we would like everyone to complete all three steps within 90 days of the training course.  If you are not able to do this, please contact me to discuss reasons and options.
```

Comment:
This is usually sent out as plain text in an email message, rather than as formatted Markdown, so URL-formatting like this won't work.